source/BankPanel.cpp

```diff
@@ -227,9 +248,12 @@ void BankPanel::Draw()
 	// Draw the "Pay All" button.
 	const Interface *interface = GameData::Interfaces().Get("bank");
 	Information info;
-	for(const Mortgage &mortgage : player.Accounts().Mortgages())
-		if(mortgage.Principal() <= player.Accounts().Credits())
-			info.SetCondition("can pay");
+	if((salariesOwed || maintenanceDue) && player.Accounts().Credits() != 0)
```

Comment:
What if player.Accounts().Credits() is negative?

Reply:
Can it be?

Reply:
Credits is a signed 64 bit integer value, so yes, in theory the number can be negative.
I expect that the number can go negative when a mission has a negative payment (but I didn't check this).
Loading a savegame with a negative number is probably also possible (but again I didn't check this).

My suggestion is to use some defensive programming here `player.Accounts().Credits() > 0`and thus handle the situation when the number is negative, even if we are not sure if the number ever actually goes negative.

Reply:
Negative payment missions can’t offer/complete if you don’t have enough credits for the negative payment. 
As with many situations in game, I wouldn’t be surprised if forcing something to be negative via save editing or text file editing caused strange results. (See how negative slowing damage causes ships to spin rapidly then fling off into space.) Save editing to have negative credits does indeed give you negative credits in game, though.